src/include/meta.h

```diff
@@ -46,6 +46,9 @@
 #define WT_METADATA_VERSION "WiredTiger version" /* Version keys */
 #define WT_METADATA_VERSION_STR "WiredTiger version string"
 
+#define WT_MIN_COMPAT_VERSION_MAJOR 3 /* Minimum version we can start on. */
```

Comment:
I also think this deserves a big comment, describing why we are enforcing a floor. It's really only because we are trying to stop MongoDB users from hurting themselves, and I don't think we'll change these numbers again for a very long time - so having a comment describing how and why they are used will be worthwhile.

Reply:
Added comment.